ds3-autogen-c/src/main/resources/templates/request-templates/SpectraS3GetRequest.ftl

```diff
@@ -0,0 +1,29 @@
+<#-- ********************************** -->
+<#-- Generate "SpectraS3 Get Request" -->
+<#--   Input: Source object             -->
+<#-- ********************************** -->
+<#list getRequests() as requestEntry>
+    <#if (requestEntry.getClassification().toString() == "spectrads3") && (requestEntry.getVerb().toString() == "GET")>
```

Comment:
... thats not what you suggested when I re-worked the CCodeGenerator and Source / Header objects... but I can re-redo it if you feel strongly.  I agree, because it would be simpler to filter it in the CCodeGenerator and pick a specific template with less logic baked in, but didn't go that route because of your previous comments